lib/CSoundBase.h

```diff
@@ -259,7 +259,7 @@ VCMI_SOUND_NAME(Dig) VCMI_SOUND_FILE(DIGSOUND.wav) \
 VCMI_SOUND_NAME(DIPMAGK) VCMI_SOUND_FILE(DIPMAGK.wav) \
 VCMI_SOUND_NAME(DISEASE) VCMI_SOUND_FILE(DISEASE.wav) \
 VCMI_SOUND_NAME(DISGUISE) VCMI_SOUND_FILE(DISGUISE.wav) \
-VCMI_SOUND_NAME(DISPELL) VCMI_SOUND_FILE(DISPELL.wav) \
+VCMI_SOUND_NAME(DISPEL) VCMI_SOUND_FILE(DISPEL.wav) \
```

Comment:
@IvanSavenko should file name change stay as it was?

Reply:
Does the change to `VCMI_SOUND_NAME` have to be reverted as well?

Reply:
Yes. 
These values refer to names of files inside H3 sound archive.
To be honest we can remove most of these values - they are kind of deprecated, with most of them now migrated to config.